docs/tutorials/projects/bedtime-songs-bot.md

```diff
@@ -5,9 +5,7 @@ weight: 60
 type: "docs"
 tags: ["mac", "app", "board", "webcam", "camera", "ml", "machine learning", "babysitter"]
 description: "Create a robot babysitter with a webcam and machine learning."
-image: "/tutorials/bedtime-songs-bot/export-screenshot.png"
 imageAlt: "Image of Tess and their computer."
```

Comment:
```suggestion
images: [ "/tutorials/bedtime-songs-bot/robot_babysitter.gif" ]
```
![robot_babysitter](https://github.com/viamrobotics/docs/assets/5212232/3b4f4df4-36a3-4e55-8b94-f957dccaedaa)

^ just videoAlt when you use videos but we do want to add an images argument with a gif in it